tests/playwright/src/model/pages/containers-page.ts

```
@@ -150,6 +152,32 @@ export class ContainersPage extends MainPage {
     });
   }
 
+  async checkAllContainers(): Promise<void> {
+    return test.step('Checks all containers', async () => {
+      let containersTable;
```

Biome (2.1.2)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)